lib/transition.js

```diff
@@ -118,28 +118,43 @@ class Transition {
       return
     }
 
+    let isPrefixed = false
+    let hasAssociatedProp = false
+
     decl.parent.each(i => {
       if (i.type !== 'decl') {
         return undefined
       }
       if (i.prop.indexOf('transition-') !== 0) {
         return undefined
       }
+      let values = list.comma(i.value)
+      // check if current Rule's transition-property comma separated value list needs prefixes
       if (i.prop === 'transition-property') {
+        values.forEach(value => {
+          let lookup = this.prefixes.add[value]
+          if (lookup && lookup.prefixes && lookup.prefixes.length > 0) {
+            isPrefixed = true
+          }
+        })
         return undefined
       }
-
-      if (list.comma(i.value).length > 1) {
-        decl.warn(
-          result,
-          'Replace transition-property to transition, ' +
-            'because Autoprefixer could not support ' +
-            'any cases of transition-property ' +
-            'and other transition-*'
-        )
+      // check if another transition-* prop in current Rule has comma separated value list
+      if (values.length > 1) {
```

Comment:
Can you replace it to simpler

```js
hasAssociatedProp = values.length > 1
```

Reply:
Not really, because a subsequent transition-* property (without a comma separated value list) might override it in a following iteration.

I could replace it with 
```js
hasAssociatedProp = hasAssociatedProp || values.length > 1
```